tests/contenttypes_tests/test_order_with_respect_to.py

```diff
@@ -1,18 +1,18 @@
-from order_with_respect_to.tests import (
-    OrderWithRespectToTests, OrderWithRespectToTests2,
+from order_with_respect_to.base_tests import (
+    BaseOrderWithRespectToTests, BaseOrderWithRespectToTests2,
 )
 
 from .models import Answer, Post, Question
 
 
-class OrderWithRespectToGFKTests(OrderWithRespectToTests):
+class OrderWithRespectToGFKTests(BaseOrderWithRespectToTests):
     Answer = Answer
     Question = Question
 
-del OrderWithRespectToTests
+del BaseOrderWithRespectToTests
```

Comment:
I think the mixin and `TestCase` approach is more common in the tests code base.